### ISSUE TITLE
[Feature] Setting recovery options on install

### ISSUE THREAD

**Issue by iamcarbon** [PeterKottas/DotNetCore.WindowsService#63]
It would be useful to be able to set the recovery options when installing the service.

**Comment by PeterKottas** [comment]
Yeah why not. I've never looked at this before so it will need some digging but I'll give it a shot when I have a chance. It will take a little while though so anybody is free to pick this up

**Comment by dasMulli** [comment]
The library used for interaction with the service management APIs has this feature in the current public version, surfaced as `WithFailureAcitons` on `ServiceDefinitionBuilder` (see a [unit test config](https://github.com/dasMulli/dotnet-win32-service/blob/26df62f9e923ef8fc2016823824c2387fdb17b00/test/DasMulli.Win32.ServiceUtils.Tests/Win32ServiceManager/ServiceUpdateTests.cs#L24-L42) for an example).
